.github/workflows/deploy-docs-preview.yaml

```
@@ -23,24 +23,52 @@ jobs:
         with:
+          preview-branch: pages-pr-previews
+          pages-base-path: docs
           source-dir: docs/.vitepress/dist
-          umbrella-dir: docs/pr-preview
+          umbrella-dir: docs/previews
           action: auto
           token: ${{ secrets.PREVIEW_TOKEN }}
+
+      - name: Comment on deploy
+        uses: marocchino/sticky-pull-request-comment@v2
+        if: steps.preview-step.outputs.deployment-action == 'deploy' && env.deployment_status == 'success'
```

actionlint (1.7.4)

property "preview-step" is not defined in object type {}

(expression)